app/models/user.rb

```diff
@@ -79,10 +79,13 @@ class User < ApplicationRecord
   ].map!(&:freeze).freeze
 
   # Include default devise modules. Others available are:
-  # :confirmable, :timeoutable and :omniauthable
+  # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :invitable, :database_authenticatable, :registerable, :uid, :lockable,
          :recoverable, :rememberable, :trackable, :validatable, :omniauthable,
          omniauth_providers: %i[google_oauth2 facebook]
+  # https://github.com/michaelbanfield/devise-pwned_password#disabling-in-test-environments
+  # TODO: might be best to reenable if we disable real network requests & stub them with https://github.com/bblimke/webmock
+  devise :pwned_password unless Rails.env.test?
```

Comment:
Disabling testing for this based on the disabling in test environments section of Devise Pwned Password an external API calls. I will reenable this if we implement [Webmock](https://github.com/bblimke/webmock) then create tests that checks that users can still register if haveIBeenPwned service was down for any reason and pwned passwords to be invalid.

Reply:
Sounds good, thanks for explaining this. It sounds like we can safely disable this since we don't' need to test for this in our tests.